hyp3_testing/helpers.py

```diff
@@ -107,3 +109,20 @@ def job_tifs(job_id, api, directory, keep=False):
             for ff in product_dir.rglob('*'):
                 ff.unlink()
             product_dir.rmdir()
+
+
+@contextmanager
+def archive_tifs(product_name, product_urls, directory, keep=False):
```

Comment:
Do we see this `archive_tifs` function being used for other golden tests? If not, I'd move it to `tests/test_opera_rtc_s1.py` alongside the `get_opera_rtc_s1_info` helper function in that file.

Reply:
Nevermind, I see it's related to `job_tifs`.